.drone.yml

```diff
@@ -1,12 +1,14 @@
 ---
+
 kind: pipeline
+type: docker
```

Comment:
Is this a new thing that must be added now?

Reply:
Linting said that I needed to add it and it will run without it. The docs shows it being there in all the examples. https://docs.drone.io/pipeline/overview/